src/odoc/etc/odoc.css

```diff
@@ -432,7 +432,11 @@ pre code {
 }
 
 div.spec, .def-doc {
-  margin-bottom: 20px;
+  margin-bottom: 10px;
```

Comment:
The margin for `.spec` is useless. It is slightly bigger than the `margin-top` of the documentation, but this should be changed instead. It is lower than the margin for `.odoc-spec` so won't have any effect there either.
I've adjusted a bit: https://github.com/jonludlam/odoc/pull/42